ArkGameExample/games/FlappyBird/FlappyBirdEntityCreator.swift

```diff
@@ -29,8 +29,9 @@ enum FlappyBirdEntityCreator {
                              linearDamping: 10,
                              impulse: impulseValue,
                              categoryBitMask: FlappyBirdPhysicsCategory.character,
-                             collisionBitMask: FlappyBirdPhysicsCategory.wall | FlappyBirdPhysicsCategory.ceiling,
-                             contactTestBitMask: FlappyBirdPhysicsCategory.none)
+                             collisionBitMask: FlappyBirdPhysicsCategory.none,
+                             contactTestBitMask: FlappyBirdPhysicsCategory.wall | FlappyBirdPhysicsCategory.ceiling | FlappyBirdPhysicsCategory.scoringArea),
```

Comment:
I think can remove the bitmask for ceiling. Although its emitted, the contact event for ceiling is unused